controller/api/destination/endpoint_translator.go

```diff
@@ -474,6 +528,18 @@ func (et *endpointTranslator) sendClientRemove(set watcher.AddressSet) {
 	}
 }
 
+func (et *endpointTranslator) altFamily(setID watcher.ID, fam corev1.IPFamily) (*watcher.ID, *watcher.Address) {
```

Comment:
I think this can be written more simply as
```
altID := setID
altID.IPFamily = fam
if addr, ok := et.availableEndpoints.Addresses[altID]; ok {
  return &altID, &addr
}
return nil, nil
```
but (see other comments) I think we can probably remove or possibly inline this function altogether.

Reply:
Right, that's better :+1: